src/pyodide-py/pyodide/_base.py

```diff
@@ -49,55 +49,38 @@ class CodeRunner:
 
     Examples
     --------
-    >>> CodeRunner("1+1", {}).run()
+    >>> CodeRunner().run("1+1")
     2
-    >>> CodeRunner("1+1;", {}).run()
+    >>> CodeRunner().run("1+1;")
     """
 
-    def __init__(self, code: str, ns: Dict[str, Any]):
+    def __init__(self, ns: Dict[str, Any] = {}):
```

Comment:
We should better make it None by default, then set to `{}` if it's None, to avoid the issue of https://stackoverflow.com/a/5592432/179127

Reply:
Of course! My mistake! Never put a mutable as default argument!